### ISSUE TITLE
Component: Slider

### ISSUE THREAD

**Issue by rmcclell** [primefaces/primeng#16214]
### Describe the bug

Prime NG that have range attribute set true and orientation set horizontal, only set the role to slider correctly on the left side while the right side is missing the slider role. Accessibility audits fail due to the role missing from the second slider

Github Code Snippet
![image](https://github.com/user-attachments/assets/fc40e575-ec9c-4497-8d61-4770fac56739)

Chrome Lighthouse audit 
![image](https://github.com/user-attachments/assets/e5de67da-24cc-47eb-8bc6-d0ec2e4b3a43)

### Environment

Local environment and in prime ng examples 

### Angular version

18.1.0

### PrimeNG version

17.9.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

20.16.0

### Browser(s)

All

### Steps to reproduce the behavior

Create a prime ng horizontal slider with [range]="true" and then run lighthouse audit for accessibility in chrome on the example create and the double slider will show some accessibility errors in the audit ran.

### Expected behavior

Range type sliders should pass accessibility audits

**Comment by rmcclell** [comment]
Pending PR https://github.com/primefaces/primeng/pull/16216
